widget/view.html

```diff
@@ -10,6 +10,6 @@ <h5 data-ng-if="config.title !==''" class="padding-left-lg margin-top-0 margin-b
 </div>
 <div data-ng-if="config.embedded===undefined" class="widget widget-container chart display-flex align-items-center justify-content" style="width: 100%; height:100%" id="world-map-{{config.wid}}">
 </div>
-<div data-ng-if="config.data.length===0" class="text-center">No information available</div>
+<div data-ng-if="config.data.length===0" class="text-center font-08em">No information available</div>
```

Comment:
can't we have these changes in css or use existing platform css to keep it consistent all the time with all components/widget?

Reply:
No this is widget specific